metrics/metrics.go

```diff
@@ -63,6 +64,10 @@ var (
 		Measure:     BlockValidationSuccess,
 		Aggregation: view.Count(),
 	}
+	BlockValidationDurationView = &view.View{
+		Measure:     BlockValidationDurationMilliseconds,
+		Aggregation: view.LastValue(),
```

Comment:
Aggregation should probably be `Sum()`, so it is possible to extract what percentage of wall time is spent on validating.

Reply:
Otherwise multiple points (for each collection interval) will have the same value which will be missleading.

Reply:
That's reasonable. Will change. 👍